content/Wrath.SoundVoiceModTemplate/{SourceName}/Main.cs

```diff
@@ -109,10 +98,11 @@ static void AddAsksListBlueprint()
             {
                 OwnerBlueprint = blueprint,
 
-                // Since the blueprint is added manually by the mod, remove the usual reference
-                // to the bank name to prevent a Wwise "already loaded" error.
-                SoundBanks = [],
-                PreviewSound = "{SourceName}_Test",
+                // Change this to match the name of your primary soundbank.
```

Comment:
Is the comment from before you used SourceName?

Also why does the whitespace look off two rows below this?

Reply:
Whitespace is presumably tabs vs space.  Edit as you see fit.

Regarding `{SourceName}`, the original version of the voice mod used the same approach as the base sound mod still does to load the soundbanks. That forcibly loaded all soundbanks on game load. That would result in Wwise spamming the log with a "bank already loaded" message if the bank name was added to the blueprint, so the name was omitted. Now it only adds the soundbank/mod folder location to the Wwise bank path and lets the game load the banks on demand based on the presence of the Asks blueprint. Thus it needs the bank name to be added to the blueprint the same as vanilla banks do. `{SourceName}_GVR_ENG` is the suggested soundbank name. `{SourceName}_Test` is the preview sound Event name (i.e. the samples played in the character creator).